### PR TITLE
updated events.java

### PR DESCRIPTION
**Pull request by steakmeat197**

### CODE REVIEW

File: src/Event.java [hunk 1]
```diff
@@ -14,11 +14,21 @@
...
-    }
+    public Event(Host host) {
+     }
+    public Host getHost(){
+        return host;
+    } 
+
+     //method
+    public void Alumni(Alumni alumni){
+        alumni = guestSpeaker;}
```
Comment (tPeltier) [inline src/Event.java:25]: the assignment on line 25 is backwards. we are assigning guestSpeaker to the incoming alumni

File: src/Event.java [hunk 2]
```diff
@@ -14,11 +14,21 @@
...
+
+        public static void main(String[] args){
```
Comment (tPeltier) [inline src/Event.java]: remove line 27. there can only be one main method per program

**Review by tPeltier — CHANGES_REQUESTED** [review]
take a look at these

File: src/Event.java (hunk 1 above)
Comment (steakmeat197) [reply src/Event.java:25]: ahhh! okay i see

**Pull request closed without merging**